test/test_jit_bailouts.py

```diff
@@ -0,0 +1,10 @@
+import sys
+sys.argv.append("--test_bailouts")
```

Comment:
This looks highly suspect to me

Reply:
One complaint that @zdevito relayed to me was that when developers see failures in one of the jobs, they don't necessarily notice all the flags passed to a test file, so this is a simple compromise to create a wrapper file that passes the option. What would you think would be a better way to reduce the confusion.